.github/workflows/bsr-push.yml

```diff
@@ -0,0 +1,21 @@
+name: Push to Buf Schema Registry
+# This workflow runs when changes to proto files are pushed to main
+# and then pushes these changes on to the Buf Schema Registry at
+# https://buf.build/evmos/evmos
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
this should only be run when we cut a new release (similar to the goreleaser action)